pkg/client/mapiserver/utils.go

```diff
@@ -39,3 +41,24 @@ func getDefaultAddr() string {
 	}
 	return "http://" + env + ":5656"
 }
+
+// SortSnapshotDisksByDateTime orders the snapshot disks with respect to date and time
+func SortSnapshotDisksByDateTime(snapshotDisks []SnapshotInfo) {
```

Comment:
Can we write unit test for this ?

Reply:
This function is already covered while testing the main function i.e ListSnapshot.